Dockerfile

```diff
@@ -1,22 +1,35 @@
-FROM ubuntu:18.04
-MAINTAINER me@mehthehedgehog.be
+FROM ubuntu:18.04 as builder
 
 RUN apt-get update && \
-    apt-get install -y sudo gcc g++ python git zlib1g-dev pciutils \
-                       vim kmod strace wget curl iproute2 tcpdump \
-                       iputils-arping iputils-clockdiff iputils-ping \
-                       iputils-tracepath inetutils-traceroute mtr && \
+    apt-get install -y --no-install-recommends \
+        sudo gcc g++ python zlib1g-dev pciutils kmod && \
```

Comment:
`sudo`, `pciutils` and `kmod` are not required for `builder`

Reply:
done